calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -2713,6 +2723,20 @@ public CalendarEvent addEvent(TimeRange range, String displayName, String descri
 			return edit;
 
 		} // addEvent
+
+		/**
+		 * @return only the groups allowed and selected
+		 */
+		private Collection<Group> groupAllowedSelect (Collection<Group> group, Collection<Group> groupAllowed) {
```

Comment:
Very minor and optional, could we look for a better name?

filterAllowedAndSelectedGroups()